cluster_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	"strconv"
```

Comment:
nit, this should be up with the std lib packages, sometimes go imports is stupid.